src/Microsoft.AspNetCore.Mvc.RazorPages/DependencyInjection/PageConventionCollectionExtensions.cs

```diff
@@ -138,6 +204,41 @@ public static PageConventionCollection AuthorizePage(this PageConventionCollecti
         public static PageConventionCollection AuthorizePage(this PageConventionCollection conventions, string pageName) =>
             AuthorizePage(conventions, pageName, policy: string.Empty);
 
+        /// <summary>
+        /// Adds a <see cref="AuthorizeFilter"/> with the specified policy to the page with the specified name.
+        /// </summary>
+        /// <param name="conventions">The <see cref="PageConventionCollection"/> to configure.</param>
+        /// <param name="areaName">The area name.</param>
+        /// <param name="pageName">The page name. This value is the relative path to <see cref="RazorPagesOptions.RootDirectory"/> 
+        /// inside the area directory e.g. <c>/Manage/Users</c>. <seealso cref="PageRouteModel.ViewEnginePath" />.</param>
+        /// <param name="policy">The authorization policy.</param>
+        /// <returns>The <see cref="PageConventionCollection"/>.</returns>
+        public static PageConventionCollection AuthorizeAreaPage(
+            this PageConventionCollection conventions,
+            string areaName,
+            string pageName,
+            string policy = "")
```

Comment:
We don't really do default argument values. Just overload it

Reply:
😿